util/ca.go

```diff
@@ -29,3 +29,15 @@ func IsSelfSigned(c *x509.Certificate) bool {
 func IsSubscriberCert(c *x509.Certificate) bool {
 	return !IsCACert(c) && !IsSelfSigned(c)
 }
+
+func IsTestableBRCertificate(cert *x509.Certificate) bool {
```

Comment:
Perhaps this should be renamed? It checks if a certificate can be used for ServerAuth, so `IsServerAuthCert` or similar?

Reply:
I can do that.